### PR TITLE
create movie detail ui pull 2

### PR DESCRIPTION
**Pull request by vu9596**
create movie detail ui pull 2
https://edu-redmine.framgia.vn/issues/3537

### CODE REVIEW

**Comment by vu9596** [comment]
ready

File: app/src/main/java/vunt/com/vn/moviedb_28/screen/movieinfo/InfoFragment.java [hunk 1]
```diff
@@ -4,18 +4,37 @@
...
+    private MovieDetailViewModel mViewModel;
+    private FragmentInfoBinding mBinding;
+    private GenresDetailAdapter mGenresDetailAdapter;
 
     @Nullable
     @Override
-    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
-                             @Nullable Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.fragment_info, container, false);
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/vunt/com/vn/moviedb_28/screen/movieinfo/InfoFragment.java]: vượt 80 kí tự

File: app/src/main/java/vunt/com/vn/moviedb_28/screen/producer/ProduceFragment.java [hunk 2]
```diff
@@ -4,18 +4,37 @@
...
+    private MovieDetailViewModel mViewModel;
+    private FragmentProduceBinding mBinding;
+    private ProduceAdapter mProduceAdapter;
 
     @Nullable
     @Override
-    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
-                             @Nullable Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.fragment_produce, container, false);
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/vunt/com/vn/moviedb_28/screen/producer/ProduceFragment.java]: tương tự

File: app/src/main/java/vunt/com/vn/moviedb_28/screen/trailers/TrailerFragment.java [hunk 3]
```diff
@@ -4,18 +4,38 @@
...
+    private MovieDetailViewModel mViewModel;
+    private FragmentTrailerBinding mBinding;
+    private TrailerAdapter mTrailerAdapter;
+
     @Nullable
     @Override
-    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
-                             @Nullable Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.fragment_trailer, container, false);
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/vunt/com/vn/moviedb_28/screen/trailers/TrailerFragment.java]: tương tự

**Comment by vu9596** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Pull request merged**